### PR TITLE
add find_max_int_id to graph engine

### PR DESCRIPTION
**Pull request by kmaehashi**
Fix https://github.com/jubatus/jubatus/issues/1149

### CODE REVIEW

File: jubatus/core/driver/graph.cpp [hunk 1]
```diff
@@ -165,6 +165,10 @@ void graph::create_edge_here(
   }
 }
 
+uint64_t graph::find_max_int_id() const {
+  graph_->find_max_int_id();
```
Comment (TkrUdagawa) [inline jubatus/core/driver/graph.cpp]: Does this function return nothing?

Comment (kmaehashi) [inline jubatus/core/driver/graph.cpp]: Fixed

**Comment by kmaehashi** [comment]
Fixed the issue, added tests and rebased.

**Comment by TkrUdagawa** [comment]
LGTM!

**Pull request merged**